docker-compose.yaml

```diff
@@ -1,25 +1,23 @@
 version: "3.9"
 
 networks:
-  ren-network: 
+  ren-network:
     name: ren-network
     driver: bridge
 
 services:
   ren:
-    container_name: ren
```

Comment:
I wouwd wike tuwu keep thiws wine, as iwt names the containew awnd makes opewations awot easiew